src/code/z_kaleido_setup.c

```diff
@@ -65,23 +65,19 @@ void KaleidoSetup_Update(PlayState* play) {
 
 void KaleidoSetup_Init(PlayState* play) {
     PauseContext* pauseCtx = &play->pauseCtx;
-    u64 temp = 0; // Necessary to match
 
     pauseCtx->state = PAUSE_STATE_OFF;
     pauseCtx->debugState = 0;
-    pauseCtx->alpha = 0;
-    pauseCtx->unk_1EA = 0;
-    pauseCtx->mainState = PAUSE_MAIN_STATE_IDLE;
-    pauseCtx->mode = 0;
-    pauseCtx->pageIndex = PAUSE_ITEM;
-
-    pauseCtx->unk_1F4 = 160.0f;
-    pauseCtx->unk_1F8 = 160.0f;
-    pauseCtx->unk_1FC = 160.0f;
-    pauseCtx->unk_200 = 160.0f;
+
+    pauseCtx->eye.x = pauseCtx->eye.y = 0.0f;
     pauseCtx->eye.z = 64.0f;
     pauseCtx->unk_1F0 = 936.0f;
-    pauseCtx->eye.x = pauseCtx->eye.y = 0.0f;
+    pauseCtx->unk_1F4 = pauseCtx->unk_1F8 = pauseCtx->unk_1FC = pauseCtx->unk_200 = 160.0f;
+
+    pauseCtx->alpha = 0;
+
+    pauseCtx->unk_1EA = pauseCtx->mainState = pauseCtx->mode = pauseCtx->pageIndex = 0;
```

Comment:
is there 0 way to salvage PAUSE_MAIN_STATE_IDLE and PAUSE_ITEM

Reply:
doesn't look like it
```suggestion
    // mainState = PAUSE_MAIN_STATE_IDLE , pageIndex = PAUSE_ITEM
    pauseCtx->unk_1EA = pauseCtx->mainState = pauseCtx->mode = pauseCtx->pageIndex = 0;
```

Reply:
Yeah, I don't think so. Having both retail/debug is revealing where certain things were chained assigned because retail and debug order them in reverse directions, so I've found a few times that chained assignments is needed to match both simultaneously